pages/feed.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { compose } from 'redux'
+import Frame from '../components/Frame'
+import Feed from '../components/Feed'
+import Marketing from '../components/Marketing'
+import withData from '../lib/apollo/withData'
+import withMe from '../lib/apollo/withMe'
+import withT from '../lib/withT'
+
+const FeedPage = ({ url, me, t }) => {
+  const meta = {
+    title: t('pages/feed/title')
+  }
+  return (
+    <Frame raw url={url} meta={meta}>
+      {me && me.id ? <Feed /> : <Marketing />}
```

Comment:
Could you port [Auth/withAuthorization](https://github.com/orbiting/publikator-frontend/blob/master/components/Auth/withAuthorization.js) and use that instead? We shouldn't make pages available under multiple urls. And a person having a deep link most likely needs to log in.